lib/yandex/translator.rb

```diff
@@ -51,13 +51,14 @@ def detect(text)
       lang == '' ? nil : lang
     end
 
-    def translate(text, *lang)
+    def translate(text, *lang, format)
```

Comment:
you can provide default value 'plain' here
```
def translate(text, *lang, format = 'plain')
```

Reply:
awesome, this is cleaner